docs/installation.rst

```diff
@@ -80,6 +80,15 @@ Installation
     $ export SECRET_KEY='<mysecretkey>'
 
 
+#. Create PostgreSQL user:
+
+   .. note::
+
+       You need to create the user to use within your project.
+       Username is definied in ``docker-compose.yml``, by default ``saleor``.
```

Comment:
This is the only mention of Docker in installation docs. Can we instead state that the username and password are extracted from `DATABASE_URL` environment variable and that in its absence they default to X and Y?